src/foam/nanos/alarming/services.jrl

```diff
@@ -41,6 +40,7 @@ p({
   "serve":true,
   "serviceScript":`
     return new foam.dao.EasyDAO.Builder(x)
+      .setLastModifiedAware(true)
```

Comment:
This is automatically added it model implements LastModifiedAware - which it does. Also, since you set this manually, this is again not explicitly needed.